airlock/policies.py

```diff
@@ -68,6 +68,37 @@ def check_can_add_file_to_request(workspace: "Workspace", relpath: UrlPath):
         )
 
 
+def can_replace_file_in_request(workspace: "Workspace", relpath: UrlPath):
+    try:
+        check_can_replace_file_in_request(workspace, relpath)
+    except exceptions.RequestPermissionDenied:
+        return False
+    return True
+
+
+def check_can_replace_file_in_request(workspace: "Workspace", relpath: UrlPath):
+    """
+    This file can replace an existing file in the request.
```

Comment:
Can you expand this comment to explain how it's different to `can_update...`; it wasn't immediately obvious to me why we need to check `can_be_added` or `can_be_updated` when replacing a file (it's the withdrawn ones, right? They're still technically on the request and need to be replaced if they're being re-added)

Reply:
added a clarifying comment, does that make sense now?